pkg/cleanup/cleanup.go

```diff
@@ -38,15 +41,28 @@ var defaultCleanupSelector = &metav1.LabelSelector{
 	},
 }
 
+type ExcludeEntry struct {
+	Group   string `yaml:"group"`
+	Version string `yaml:"version"`
+	Kind    string `yaml:"kind"`
+	Name    string `yaml:"name"`
+}
+
+type groupVersionKindName struct {
+	Gvk  schema.GroupVersionKind
+	Name string
+}
```

Comment:
Same as the velero plugin here as well: Implement `matches` directly on `ExcludeEntry` and then you won't need `groupVersionKindName` anymore.